### PR TITLE
Add topics for `#media-processing` channel

### LINKED ISSUE #1052: Add topics for the new `#media-processing` channel
## Description

I think it would be nice to have some topics for the new `#media-processing` channel created 5/3/2022 for the `.topic` command.

## Reasoning

Well, all the other topical channels have topics so it only makes sense to have some for this channel as well.

## Proposed Implementation

Some topics I have in mind currently:
- Which Python package do you prefer for media processing?
- What exactly is media processing and how can Python be used for it?
- What are some ways you could create and manipulate a video using Python?

Can't think of much else at the moment. If anyone else has any ideas or has constructive criticism for the ones I mentioned above, please post a comment below. Thanks!

## Would you like to implement this yourself?

- [x] I'd like to implement this feature myself
- [ ] Anyone can implement this feature

### PR DESCRIPTION
**Pull request by DMFriends**
## Relevant Issues
Closes #1052 

## Description

Added topics for the `.topic` command for the recently created `#media-processing` channel

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read and agree to the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

**Comment by Recursive-Error** [comment]
Not sure if it helps, but here are a few additional ideas:

- Any interesting progress to share?
- Any cool trick or optimization you are proud of?
- What is your favorite media processing related algorithm?
- What is the worst outcome for your generation or processing?
- What is the most surprising result you got?
- What is your most advanced media processing related achievement?
- Do you use a lot of math in your media processing related projects?
- Can a computer truly generate art?
- Can a computer truly understand art?
- What is a good project to get started?

**Comment by DMFriends** [comment]
Oh wow, those are actually pretty awesome, @Recursive-Error , thank you so much! However, I would tweak some of them but I don't really have the time right now to write out exactly what I think needs to be tweaked, but I could do it later today if you are interested.

File: bot/resources/utilities/py_topics.yaml [hunk 1]
```diff
@@ -109,6 +109,15 @@
     - What books or tutorials would you recommend for game-development beginners?
     - What made you start developing games?
 
+# media-processing
+971142229462777926:
+    - Which Python package/library do you prefer for media processing?
+    - What exactly is media processing and how can Python be used for it?
```
Comment (dawnofmidnight) [inline bot/resources/utilities/py_topics.yaml:115]: I personally think that this question would be better fit for the channel description/a pin.

Comment (DMFriends) [reply bot/resources/utilities/py_topics.yaml:115]: I can see that, let's do a pin. Would you mind if I made it?

Comment (DMFriends) [reply bot/resources/utilities/py_topics.yaml:115]: Something like:

Some common questions related to media processing:
- Which Python package/library do you prefer for media processing?
- What exactly is media processing and how can Python be used for it?

Comment (dawnofmidnight) [reply bot/resources/utilities/py_topics.yaml:115]: > I can see that, let's do a pin. Would you mind if I made it?

I think Zig's message in the pins answers the question about "What exactly is media processing and how can Python be used for it?" already. (that's the one I meant to highlight, but GitHub also showed the other one.

Comment (DMFriends) [reply bot/resources/utilities/py_topics.yaml:115]: Oh, so
* Which Python package/library do you prefer for media processing?
is fine?

**Comment by minalike** [comment]
The question of "preferred library for media processing" is too broad to be conducive to discussion. Same with "what is media processing."

I recommend you accept most if not all of recursive_error's suggested topics. 

I especially like the suggestions of whether computers can generate or understand art.

File: bot/resources/utilities/py_topics.yaml [hunk 2]
```diff
@@ -109,6 +109,17 @@
...
 
+# media-processing
+971142229462777926:
+    - Where do you start with media processing? What is a good beginner project for first-timers in media processing?
+    - What are some ways you could manipulate an image or a video using Python?
+    - What is your favorite / a good algorithm to use for manipulating images and videos with Python?
+    - What is the most surprising result you have gotten after manipulating an image or a video with Python?
+    - What is the worst outcome you have gotten after manipulating an image or a video with Python?
+    - What is your most advanced media processing related achievement?
+    - Do you know any cool tricks for manipulating an image or a video with Python?
```
Comment (vivekashok1221) [inline bot/resources/utilities/py_topics.yaml:120]:
```suggestion
    - Do you know any cool tricks or optimizations for manipulating an image or a video with Python?
```

File: bot/resources/utilities/py_topics.yaml [hunk 3]
```diff
@@ -109,6 +109,17 @@
...
+    - Can a computer truly generate and understand art?
```
Comment (vivekashok1221) [inline bot/resources/utilities/py_topics.yaml:121]:
```suggestion
    - Can a computer truly generate and/or understand art?
```

**Review by vivekashok1221 — CHANGES_REQUESTED** [review]
Thanks for taking on this task.
Just some personal suggestions.

Also you could perhaps replace "image or video" at places to "media" so that audio/sound is also included but I'm not sure how it'd sound/if it'd sound better.

**Review by vivekashok1221 — APPROVED** [review]
lgtm.

File: bot/resources/utilities/py_topics.yaml [hunk 4]
```diff
@@ -109,6 +109,17 @@
     - What books or tutorials would you recommend for game-development beginners?
     - What made you start developing games?
 
+# media-processing
+971142229462777926:
+    - Where do you start with media processing? What is a good beginner project for first-timers in media processing?
+    - What are some ways you could manipulate media using Python?
+    - What is your favorite / a good algorithm to use for manipulating media with Python?
```
Comment (minalike) [inline bot/resources/utilities/py_topics.yaml:116]:
```suggestion
    - What is your favorite algorithm for manipulating media with Python?
```

**Review by minalike — CHANGES_REQUESTED** [review]

**Review by minalike — APPROVED** [review]

**Review by HassanAbouelela — APPROVED** [review]
Everything looks fine from a technical and grammatical aspect.

**Pull request merged**